concourse/Dockerfile

```diff
@@ -1,14 +1,23 @@
 FROM concourse/concourse
 
-ADD download-keys.sh /opt/download-keys.sh
+# This is the release of Vault to pull in.
+ENV VAULT_VERSION=0.9.0
```

Comment:
Can you parameterize the Vault version like done in this Dockerfile:
https://github.com/skyscrapers/docker-images/blob/master/packer/Dockerfile#L2
We can trigger new builds in Concourse, based on a Github release then.
https://github.com/concourse/github-release-resource

Reply:
The vault version we use here is not that relevant, as it's only used to fetch the token. The main vault implementation remains in Concourse, so I don't think we need to build new images on new Vault releases.

Reply:
It is a case like this that keeps us in requiring manual code changes when new versions become available upstream. I'm trying to advocate even more automation in our CI/CD processes, but as long as we hardcode versions, we are not improving.

Reply:
I've set it as an ARG, can you review again?